README.md

```diff
@@ -15,9 +15,7 @@
 <div align="left">
 
 OML is a PyTorch-based framework to train and validate the models producing high-quality embeddings.
-
-## FAQ
-
+\n## FAQ\n
```

Comment:
what is the idea of changing the readme in this PR? (by the way, if you want to change Readme, please, check an instruction in a contribution guide)

Reply:
It was done unintentionally. I didn't notice when it happened. I will revert the changes in README.md.